packages/graphiql/src/index.tsx

```diff
@@ -88,10 +88,64 @@ const medplumPlugin: GraphiQLPlugin = {
   ),
 };
 
+// Parse the search string to get url parameters.
+const search = window.location.search;
```

Comment:
Lines 94-102 can be simplified if you use the `URLSearchParams` class: https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/URLSearchParams

That way we don't have to do our own string parsing.

For example:

```ts
const searchParams = new URLSearchParams(location.href.search);
const parameters: FetcherParams = {
  query: searchParams.get('query'),
  variables: searchParams.get('variables'),
  operationName : searchParams.get('operationName'),
};
```